src/turnkeyml/cli/cli.py

```diff
@@ -125,9 +131,11 @@ def main():
         "-i",
         "--input-files",
         nargs="+",
-        help="One or more script (.py), ONNX (.onnx), or input list (.txt) files to be benchmarked",
+        help="One or more inputs that will be evaluated by the tool sequence "
+        "(e.g., script (.py), ONNX (.onnx), turnkey build state (state.yaml), "
+        "input list (.txt) files)",
         type=lambda file: _check_extension(
-            ("py", "onnx", "txt"), file, parser.error, tool_classes
+            ("py", "onnx", "txt", "yaml"), file, parser.error, tool_classes
```

Comment:
Would be nice if load-build could simply point to a build directory rather than the yaml file, but I can see how implementing that would be non-trivial.
No action needed here.

Reply:
+1

Reply:
Yeah I definitely agree and though about it. It was just too much work to enable... our API is called "evaluate *files*()" and I would have needed to boil the ocean to support non-file inputs.